spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-kubernetes/src/test/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/KubernetesModuleDeployerIT.java

```diff
@@ -0,0 +1,110 @@
+package org.springframework.cloud.dataflow.module.deployer.kubernetes;
+
+import static org.junit.Assert.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.cloud.dataflow.core.ArtifactCoordinates;
+import org.springframework.cloud.dataflow.core.ModuleDefinition;
+import org.springframework.cloud.dataflow.core.ModuleDeploymentId;
+import org.springframework.cloud.dataflow.core.ModuleDeploymentRequest;
+import org.springframework.cloud.dataflow.module.ModuleStatus;
+import org.springframework.cloud.dataflow.module.deployer.ModuleDeployer;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = KubernetesModuleDeployerConfiguration.class)
+public class KubernetesModuleDeployerIT {
```

Comment:
We'll want to make this conditional to the fact that there is a kubernestes cluster running where we expect it to.
See for example a JUnit rule to only run ITs thats require redis when redis is available: https://github.com/spring-cloud/spring-cloud-stream/blob/7023e161967da795ab5f94fde6b4a5964d8c1aa2/spring-cloud-stream-test-support-internal/src/main/java/org/springframework/cloud/stream/test/junit/redis/RedisTestSupport.java

I think this can go in a different PR though